contest/contributions/antoine-soetewey-covid-19-in-belgium-is-it-over-yet.yml

```
@@ -0,0 +1,16 @@
+# Contribution's title, author, abstract
+title: 'COVID-19 in Belgium: is it over yet?'
+author: Antoine Soetewey
+abstract: |
+  This article shows the evolution of hospital admissions and number of new confirmed cases due to COVID-19 in Belgium.
+
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: "Evolution of hospital admissions and number of new confirmed cases due to COVID-19 in Belgium"
```

Shorter `menu_entry`, same as `title`
```suggestion
menu_entry: 'COVID-19 in Belgium: is it over yet?' 
```